lib/license_scout/overrides.rb

```diff
@@ -671,6 +671,7 @@ def default_overrides
         ["debug", "MIT", ["https://raw.githubusercontent.com/visionmedia/debug/master/LICENSE"]],
         ["performance-now", "MIT", ["https://raw.githubusercontent.com/braveg1rl/performance-now/master/license.txt"]],
         ["cli-table", "MIT", ["README.md"]],
+        ["process", "MIT", ["https://raw.githubusercontent.com/defunctzombie/node-process/master/LICENSE"]],
```

Comment:
@danielsdeleo It was def a zombie comment, but also an odd situation here because we pull in both 0.11.10 and 0.5.2, where 0.11.10 has license info in the package.json so it's cool. something pulled in 0.5.2 which is :sadpanda: